app/assets/javascripts/application.js

```diff
@@ -20,13 +20,113 @@ const closeModalListener = (element) => {
     let closeButton = document.getElementById(elementToObserve)
     let modalElement = document.getElementById('modal')
 
-    if (closeButton && modalElement){
-        closeButton.addEventListener('click', ()=>{
+    if (closeButton && modalElement) {
+        closeButton.addEventListener('click', () => {
             modalElement.remove()
         })
     }
 }
 
-document.addEventListener('turbolinks:load', ()=>{
-    App.windowContent = document.getElementById('content')
+const stripeTokenHandler = (token) => {
+    var form = document.getElementById('payment-form');
```

Comment:
Use `let` instead of `var` and if its not a variable that will be reassigned value it should be `const`